tsconfig.json

```diff
@@ -7,10 +7,10 @@
     "emitDeclarationOnly": true,
     "target": "es2015",
 
-    "lib": ["esnext"],
+    "lib": ["esnext", "dom"],
```

Comment:
This is not correct for the majority of packages, and almost all packages inherit from this config.

Reply:
yah that was dumb, sorry for being a n00b :cry: